### DISCUSSION TITLE
Only index file under pages folder accessible in deployed app

### DISCUSSION THREAD

**Discussion by dimassibassem** [hannoeru/vite-plugin-pages#386] [Q&A]
### Describe the bug

The issue I'm encountering is that after deploying the app on Vercel, I'm only able to access the index file located under the pages folder. This means that the rest of the app's pages are not accessible and returning 404.

### Reproduction

https://github.com/dimassibassem/modelerUI/tree/master/Client

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    Memory: 2.47 GB / 15.86 GB
  Binaries:
    Node: 18.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 8.19.3 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (110.0.1587.56)
    Internet Explorer: 11.0.22000.120
```

### Used Package Manager

pnpm

### Validations

- [x] Follow the [Code of Conduct](https://github.com/hannoeru/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the docs.
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A please open a GitHub Discussion.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hannoeru** [comment]
https://vercel.com/docs/concepts/projects/project-configuration#routes/advanced/spa-fallback

**Comment by coderbaozi** [comment]
ennnn,I had meet same situation. qaq

**Comment by sistematico** [comment]
Sorry for hi-jacking thread, I have same issue with Github Pages, no clues at all.
